src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsSummary.scala

```diff
@@ -1,37 +1,37 @@
 package org.globalforestwatch.summarystats.firealerts
 
 import cats.implicits._
-import geotrellis.contrib.polygonal.CellVisitor
 import geotrellis.raster._
+import geotrellis.raster.summary.GridVisitor
 import org.globalforestwatch.summarystats.Summary
 import org.globalforestwatch.util.Util.getAnyMapValue
 import org.globalforestwatch.util.{Geodesy, Mercantile}
 
 /** LossData Summary by year */
 case class FireAlertsSummary(stats: Map[FireAlertsDataGroup, FireAlertsData] =
-                             Map.empty,
-                             kwargs: Map[String, Any])
+                             Map.empty)
   extends Summary[FireAlertsSummary] {
 
   /** Combine two Maps and combine their LossData when a year is present in both */
   def merge(other: FireAlertsSummary): FireAlertsSummary = {
     // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
-    FireAlertsSummary(stats.combine(other.stats), kwargs)
+    FireAlertsSummary(stats.combine(other.stats))
   }
 }
 
 object FireAlertsSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
 
   implicit val mdhCellRegisterForFireAlertsRaster
-    : CellVisitor[Raster[FireAlertsTile], FireAlertsSummary] =
-    new CellVisitor[Raster[FireAlertsTile], FireAlertsSummary] {
+    : GridVisitor[Raster[FireAlertsTile], FireAlertsSummary] =
+      new GridVisitor[Raster[FireAlertsTile], FireAlertsSummary] {
+      private var acc: FireAlertsSummary = new FireAlertsSummary()
 
-      def register(raster: Raster[FireAlertsTile],
-                   col: Int,
-                   row: Int,
-                   acc: FireAlertsSummary): FireAlertsSummary = {
+      def result: FireAlertsSummary = acc
 
+      def visit(raster: Raster[FireAlertsTile],
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)